src/test/java/xyz/groundx/caver_ext_kas/kas/kip37/KIP37Test.java

```diff
@@ -0,0 +1,1812 @@
+        callback.checkResponse();
+
+        Thread.sleep(5000);
+    }
+
+    @Test
+    public void transferAsync_multipleToken_hexStringType() throws ApiException, TransactionException, IOException, InterruptedException, ExecutionException {
+        Account account = caver.kas.wallet.createAccount();
+        String[] tokenIdArray = new String[]{Numeric.toHexStringWithPrefix(CREATED_TOKEN_ID_1), Numeric.toHexStringWithPrefix(CREATED_TOKEN_ID_2)};
+        String[] amountArray = new String[]{"0x1", "0x1"};
+
+        Kip37TransactionStatusCallback callback = new Kip37TransactionStatusCallback();
+
+        caver.kas.kip37.transferAsync(testContractAddress, deployerAddress, deployerAddress, account.getAddress(), tokenIdArray, amountArray, callback);
+        callback.checkResponse();
+
+        Thread.sleep(5000);
+    }
+
+    @Test
+    public void pauseToken_BigIntegerType() throws ApiException, TransactionException, IOException, InterruptedException {
+        BigInteger tokenId = BigInteger.valueOf(30000);
+        createToken(testContractAddress, tokenId, BigInteger.ONE, "https://token-cdn-domain/0x02.json");
+        Thread.sleep(5000);
+
+        Kip37TransactionStatusResponse response = caver.kas.kip37.pauseToken(testContractAddress, tokenId);
+        getReceipt(caver, response.getTransactionHash());
+
+        Thread.sleep(5000);
+    }
+
+    @Test
+    public void pauseToken_hexStringType() throws ApiException, TransactionException, IOException, InterruptedException {
+        BigInteger tokenId = BigInteger.valueOf(30001);
+        String tokenIdHex = Numeric.toHexStringWithPrefix(tokenId);
+        createToken(testContractAddress, tokenId, BigInteger.ONE, "https://token-cdn-domain/0x02.json");
+        Thread.sleep(5000);
+
+        Kip37TransactionStatusResponse response = caver.kas.kip37.pauseToken(testContractAddress, tokenIdHex);
+        getReceipt(caver, response.getTransactionHash());
+
+        Thread.sleep(5000);
+    }
+
+    @Test
+    public void pauseToken_BigIntegerType_withSender() throws ApiException, TransactionException, IOException, InterruptedException {
+        BigInteger tokenId = BigInteger.valueOf(30002);
+        createToken(testContractAddress, BigInteger.valueOf(30002), BigInteger.ONE, "https://token-cdn-domain/0x02.json");
+        Thread.sleep(5000);
+
+        Kip37TransactionStatusResponse response = caver.kas.kip37.pauseToken(testContractAddress, tokenId, deployerAddress);
+        getReceipt(caver, response.getTransactionHash());
+
+        Thread.sleep(5000);
+    }
+
+    @Test
+    public void pauseToken_hexStringType_withSender() throws ApiException, TransactionException, IOException, InterruptedException {
+        BigInteger tokenId = BigInteger.valueOf(30003);
+        String tokenIdHex = Numeric.toHexStringWithPrefix(tokenId);
+        createToken(testContractAddress, tokenId, BigInteger.ONE, "https://token-cdn-domain/0x02.json");
+        Thread.sleep(150000);
```

Comment:
Does this test case take a long time than other cases?
It's not a big issue, but other test cases use `5000` except this test case.

Reply:
Okay, I'll fix it. Thanks @aeharvlee.